front/cypress/support/commands.ts

```diff
@@ -1706,6 +1734,48 @@ function apiCreateSurveyResponse(
         },
       },
     });
+  };
+
+  if (jwt) {
+    return makeRequest(jwt);
+  } else {
+    return cy.apiLogin(email, password).then((response) => {
+      const jwt = response.body.jwt;
+      return makeRequest(jwt);
+    });
+  }
+}
+
+function uploadSurveyImageQuestionImage(base64: string) {
+  return cy.apiLogin('admin@citizenlab.co', 'democracy2.0').then((response) => {
+    const adminJwt = response.body.jwt;
+
+    return cy.request({
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${adminJwt}`,
+      },
+      method: 'POST',
+      url: 'web_api/v1/custom_field_option_images',
+      body: {
+        image: { image: base64 },
+      },
+    });
+  });
+}
+
+function apiGetSurveySchema(phaseId: string) {
+  return cy.apiLogin('admin@citizenlab.co', 'democracy2.0').then((response) => {
```

Comment:
Would it make sense to apply the same optimization as you did [here](https://github.com/CitizenLabDotCo/citizenlab/pull/7348/files#diff-2ae9fb29e6ec0538dfc12563236236298f64d5c731764b81e4ec45dae35b304dR1739) by checking if `jwt` is already present?